test/solidity-by-example/specs/ApiSession.Solidity-by-Example.receipts.spec.ts

```diff
@@ -42,7 +41,6 @@ describe("ApiSession.Solidity-by-Example.Receipts", () => {
   it("uploading a contract should generate appropriate receipts regardless if constructor-event logs are of interest or not", async () => {
     const expectedTransactionSources = [
       FileCreateTransaction,
-      FileAppendTransaction,
```

Comment:
Bumping file-chunk size to `2048` means we actually don't need an append for this contract, so we drop it.